deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -76,6 +76,10 @@ data:
             hosts:
               - 'kube-state-metrics:8080'
             period: 10s
+            # If `https` is used to access `kube-state-metrics`, then to all `kubernetes.state_*` datasets should be added:
```

Comment:
KSM does not have built in authentication/authorization and this can only be performed with third party [tools](https://github.com/kubernetes/kube-state-metrics/issues/1714#issuecomment-1092629353). I think we should not set the kubernetes bearer token and certificate authorities as the default values here because it can be misleading. @gizas WDYT?

Reply:
rephrased this comment - https://github.com/elastic/elastic-agent/pull/708/commits/2e153727d87366943e938aa256fdbf774a375aea